src/main/frontend/common/i18n/translations.ts

```diff
@@ -22,7 +22,9 @@ export class Translations {
     if (message != null) {
       return message;
     }
-    console.debug(`Translation for ${key} not found, using fallback`);
+    if (!process.env.JEST_WORKER_ID) {
+      console.debug(`Translation for ${key} not found, using fallback`);
```

Comment:
loads of spam otherwise, although could mock the response maybe, this is simpler for now though